pkg/planner/core/planbuilder.go

```diff
@@ -1216,6 +1216,19 @@ func getPossibleAccessPaths(ctx sessionctx.Context, tableHints *hint.TableHintIn
 	if len(available) == 0 {
 		available = append(available, tablePath)
 	}
+
+	// If all available paths are Multi-Valued Index, it's possible that the only multi-valued index is inapplicable,
+	// so that the table paths are still added here to avoid failing to find any physical plan.
+	allMVIIndexPath := true
+	for _, availablePath := range available {
+		if !isMVIndexPath(availablePath) {
+			allMVIIndexPath = false
+		}
+	}
+	if allMVIIndexPath {
+		available = append(available, tablePath)
```

Comment:
Here is how I fixed the "Can't find a proper physical plan for this query". 

Is the former `if len(available) == 0` only an optimization to explore less paths, or it's necessary to ensure the hint works properly? I saw some paths are added to `preferredPath` according to the `.Force` field later, but I'm not sure whether it's really a hard restriction, or just some soft preference :thinking: .

@AilinKid PTAL

Reply:
Oops, I realized that the index merge access path doesn't have `.Force` :facepalm: . Let me think about it.

Reply:
Oops. I'll need to remove these plan when after finding a valid index merge based in multi- valued index.

Reply:
Done

Reply:
LGTM here